app/src/main/res/drawable/thumb_mercadolibre.xml

```diff
@@ -0,0 +1,24 @@
-6.47 -9.01,-8.5c-0.6,-0.5 -1.23,-0.95 -1.84,-0.95c-0.3,0 -0.57,0.11 -0.78,0.33c-0.28,0.31 -0.47,0.87 0.22,1.79c0.29,0.38 0.62,0.69 0.62,0.69l9,10.1c0.08,0.09 1.86,2.2 0.2,4.31l-0.32,0.4c-0.27,0.3 -0.56,0.57 -0.84,0.8c-1.54,1.26 -3.58,1.4 -4.4,1.4c-0.43,0 -0.85,-0.04 -1.22,-0.1c-0.89,-0.16 -1.48,-0.41 -1.77,-0.75l-0.11,-0.11c-0.49,-0.51 -5.03,-5.14 -8.78,-8.27c-0.49,-0.42 -1.11,-0.94 -1.75,-0.94c-0.32,0 -0.6,0.12 -0.81,0.36c-0.74,0.81 0.37,2.03 0.84,2.48l7.67,8.46c-0.01,0.08 -0.1,0.25 -0.29,0.52c-0.28,0.38 -1.2,1.31 -3.99,1.66c-0.33,0.04 -0.68,0.06 -1.02,0.06c-2.87,0 -5.93,-1.39 -7.51,-2.23c0.72,-1.52 1.09,-3.19 1.09,-4.86c0,-6.32 -5.12,-11.45 -11.44,-11.46c-0.14,0 -0.28,0 -0.41,0.01c0.21,-2.88 -0.2,-8.34 -5.8,-10.74c-1.61,-0.7 -3.22,-1.05 -4.78,-1.05c-1.22,0 -2.4,0.21 -3.51,0.64c-1.16,-2.26 -3.09,-3.9 -5.6,-4.76c-1.39,-0.48 -2.77,-0.72 -4.11,-0.72c-2.33,0 -4.48,0.69 -6.4,2.05c-1.84,-2.28 -4.61,-3.64 -7.53,-3.64c-2.56,0 -5.01,1.02 -6.83,2.83c-2.39,-1.82 -11.86,-7.84 -37.21,-13.6c-1.23,-0.28 -4.04,-1.08 -5.77,-1.59c-0.28,1.37 -0.5,2.76 -0.65,4.16c0,0 4.68,1.12 5.6,1.33c25.9,5.75 34.46,11.73 35.9,12.86c-0.49,1.18 -0.74,2.45 -0.75,3.72c0,5.35 4.35,9.71 9.7,9.71c0.6,0 1.2,-0.05 1.78,-0.16c0.8,3.94 3.38,6.92 7.32,8.46c1.15,0.44 2.32,0.67 3.46,0.67c0.74,0 1.48,-0.09 2.21,-0.28c0.72,1.85 2.36,4.15 6.02,5.64c1.28,0.52 2.56,0.79 3.81,0.79c1.02,0 2.02,-0.18 2.96,-0.53c1.75,4.27 5.92,7.1 10.58,7.1c3.08,0 6.04,-1.25 8.2,-3.47c1.85,1.03 5.75,2.88 9.69,2.89c0.51,0 0.99,-0.04 1.47,-0.09c3.91,-0.5 5.74,-2.02 6.57,-3.22c0.15,-0.21 0.29,-0.43 0.4,-0.66c0.92,0.26 1.94,0.48 3.1,0.48c2.14,0 4.19,-0.73 6.26,-2.24c2.04,-1.47 3.48,-3.58 3.69,-5.37c0,-0.02 0.01,-0.05 0.01,-0.08c0.69,0.14 1.39,0.22 2.1,0.22c2.2,0 4.36,-0.69 6.43,-2.04c3.99,-2.62 4.68,-6.03 4.62,-8.27c0.7,0.15 1.41,0.22 2.12,0.22c2.07,0 4.09,-0.62 6.03,-1.86c2.47,-1.58 3.96,-4 4.19,-6.82c0.16,-1.92 -0.32,-3.84 -1.33,-5.5c6.68,-2.88 21.96,-8.45 39.95,-12.51c-0.11,-1.4 -0.31,-2.77 -0.56,-4.14C173.8,62.97 157.55,70 153.48,71.81zM92.63,106.99c-4.23,0 -7.66,-3.29 -7.82,-7.48c-0.01,-0.36 -0.05,-1.31 -0.86,-1.31c-0.33,0 -0.62,0.2 -0.95,0.5c-0.93,0.86 -2.11,1.74 -3.84,1.74c-0.78,0 -1.64,-0.18 -2.53,-0.55c-4.46,-1.81 -4.52,-4.88 -4.34,-6.11c0.05,-0.33 0.07,-0.67 -0.16,-0.94l-0.28,-0.25h-0.28c-0.23,0 -0.46,0.09 -0.78,0.32c-1.29,0.91 -2.53,1.35 -3.79,1.35c-0.69,0 -1.4,-0.14 -2.12,-0.41c-5.88,-2.29 -5.41,-7.84 -5.13,-9.5c0.04,-0.34 -0.04,-0.6 -0.26,-0.78l-0.42,-0.34l-0.39,0.37c-1.15,1.1 -2.64,1.71 -4.22,1.71c-3.37,0 -6.1,-2.74 -6.1,-6.1c0,-3.37 2.74,-6.1 6.11,-6.1c3.04,0 5.63,2.29 6.02,5.32l0.21,1.64l0.9,-1.39c0.1,-0.16 2.57,-3.89 7.1,-3.89c0.86,0 1.75,0.14 2.65,0.42c3.62,1.1 4.23,4.38 4.32,5.74c0.06,0.8 0.63,0.84 0.74,0.84c0.31,0 0.54,-0.2 0.7,-0.37c0.68,-0.71 2.17,-1.9 4.49,-1.9c1.06,0 2.2,0.26 3.36,0.76c5.74,2.46 3.14,9.76 3.11,9.84c-0.49,1.21 -0.51,1.74 -0.05,2.05l0.23,0.11h0.17c0.26,0 0.58,-0.11 1.11,-0.29c0.78,-0.27 1.96,-0.68 3.06,-0.67l0,0c4.32,0.05 7.84,3.57 7.84,7.84C100.47,103.48 96.95,106.99 92.63,106.99z"
```

Comment:
If it's easy to do, maybe you could split this path in several smaller pieces. With 4770 characters it's rather long and for performance reasons it's recommended to keep it under 800 characters. If it takes too long, we will live with it the way it is and maybe optimize it in the future.

Reply:
This one I don't know how to do heh

Reply:
This one is a hard one. We will leave it as is for now.